ae-ease-to-gsap-customease.jsx

```diff
@@ -248,25 +248,15 @@
 		var endInterpolation = property.keyInInterpolationType(endIndex);
 
 		if (startInterpolation === KeyframeInterpolationType.LINEAR &&
-			endInterpolation === KeyframeInterpolationType.LINEAR) {
+			endInterpolation !== KeyframeInterpolationType.HOLD) {
```

Comment:
I don't think this is correct. This would cause a linear-bezier pair to be calculated as a linear-linear pair.